app/templates/eventNav.html

```diff
@@ -18,13 +18,20 @@
         {% if g.current_user.isAdmin %}
             {{ eventheader(page_title, event, 'large', isNewEvent) }}
         {% endif %}
-        {% set alertClass = "alert-danger" if isPastEnd else "alert-warning"%}
-        {% set display = "" if isPastEnd or event.isCanceled else "d-none"%}
+        {% set isInProg = not event.isPastEnd and event.isPastStart %}
+        {% set display = "" if eventPast or event.isCanceled or isInProg else "d-none"%}
+        {% set alertClass = ("alert-success" if isInProg else ( "alert-danger" if eventPast else "alert-warning")) %}
+        {% set event_year = event.endDate.strftime("%Y") | int %}
+        {% set currentYear = g.currentDateTime.year | int %}
+        {% set formatted_date = event.endDate.strftime("%B %d %Y") if currentYear > event_year else event.endDate.strftime("%B %d") %}
+
         <div class = "{{alertClass}} {{display}} mb-2 pb-2 pt-2" role="alert">
-        {% if isPastEnd and event.isCanceled %}
+        {% if eventPast and event.isCanceled %}
             This is a past event and it was canceled.
-        {% elif isPastEnd %}
-            This is a past event.
+        {% elif isInProg %}
+            This event is happening right now 
+        {% elif eventPast %}
+            This event ended on {{ formatted_date }} at {{ event.timeEnd.strftime("%I:%M %p") }}
```

Comment:
End sentences with a period, please. Or at least match what the other messages had, since we don't always obey grammar rules on websites.